.github/workflows/build.yml

```
@@ -82,7 +82,7 @@ jobs:
       - name: Build (PR)
         if: github.ref != 'refs/heads/master'
         run: |
-          mvn -B -e -DskipTests=true install
+          mvn -T3 -B -e -DskipTests=true install -DargLine="-XX:MaxRAMPercentage=90.0"
```

Setting -DargLine here isn't useful, since we're skipping tests.
```suggestion
          mvn -T3 -B -e -DskipTests=true install"
```